src/main.cpp

```diff
@@ -12,17 +12,18 @@ int main()
     intel.drawBoard();
 
     do{
-//        std::cout << "Enter X position : ";
-//        std::cin >> x;
-//        std::cout << "\n ";
+        //        std::cout << "Enter X position : ";
+        //        std::cin >> x;
+        //        std::cout << "\n ";
 
-//        std::cout << "Enter Y position : ";
-//        std::cin >> y;
-//        std::cout << "\n ";
+        //        std::cout << "Enter Y position : ";
+        //        std::cin >> y;
+        //        std::cout << "\n ";
```

Comment:
you must remove this place. If comments not contains some info, it must be removed.